### ISSUE TITLE
Date Filter Not Working

### ISSUE THREAD

**Issue by fishytodd** [WCGA/Marine-Planner#13]
I'm not sure the date filter is working properly...For example, I selected cigar tips then changed the dates, updating the filter each time. The debris number in three places (LA area, north coast of OR, Puget Sound) never changed, but it did change in other areas. The same thing happens for cigarette butts. - Briana

**Comment by fishytodd** [comment]
I also got some strange behavior as I changed filters. I filtered for “cigarette lighters” and got numbers up and down the coast. Then I added a filter for a narrow date range (in addition to “cigarette lighters”) I got results only in LA. Then when I broadened the date range back out again the number went up but still only showed results in LA. When I removed all filters and hit “update filter” it shows only numbers for LA, WA, and OR. I attached a screen shot of this last situation. - karen
